src/_locales/hu/messages.json

```diff
@@ -0,0 +1,701 @@
+        "message": "A láthatatlan követés mindenféle módon történhet, a reklámok csak a jéghegy csúcsát jelentik. A Privacy Badger elküldi a <a href='https://globalprivacycontrol.org/' target='_blank'>globális adatvédelmi szabályozás jelzését</a>, hogy lemondjon az Ön nevében az adatmegosztásról és az adateladásról, valamint a <a href='https://www.eff.org/issues/do-not-track' target='_blank'>ne kövessen jelzést</a>, hogy közölje a vállalatokkal, hogy ne kövessék Önt. Ha azok figyelmen kívül hagyják a kéréseit, akkor a Privacy Badger meg fogja tanulni tiltani azokat – legyenek azok hirdetők vagy bármilyen fajta követők.",
+        "description": "Intro page paragraph."
+    },
+    "report_close": {
+        "message": "Bezárás",
+        "description": ""
+    },
+    "import_user_data": {
+        "message": "Felhasználói adatok importálása",
+        "description": ""
+    },
+    "options_domain_filter_dnt": {
+        "message": "DNT-kompatibilis",
+        "description": "Dropdown control setting on the Tracking Domains options page tab."
+    },
+    "popup_help_button": {
+        "message": "Súgó",
+        "description": "Tooltip that comes up when you hover over the question mark button in the upper right corner of the popup."
+    },
+    "options_domain_filter_allow": {
+        "message": "engedélyezve",
+        "description": "Dropdown control setting on the Tracking Domains options page tab."
+    },
+    "intro_donate_heading": {
+        "message": "Az adatvédelem csapatsport!",
+        "description": "Part of the 'donate' section on the intro page."
+    },
+    "show_tracking_domains_message": {
+        "message": "Itt nem kellene semmit sem módosítania.",
+        "description": "Shown above the acknowledgement checkbox required to reveal the tracking domains list on the options page. This is the second paragraph; the first paragraph is the message under the \"intro_not_an_adblocker_paragraph\" key."
+    },
+    "options_domain_filter_all": {
+        "message": "összes",
+        "description": "Dropdown control setting on the Tracking Domains options page tab."
+    },
+    "report_cancel": {
+        "message": "Mégse",
+        "description": ""
+    },
+    "tooltip_block": {
+        "message": "Húzza a csúszkát balra egy tartomány tiltásához",
+        "description": ""
+    },
+    "options_enable_dnt_checkbox": {
+        "message": "A „<a href='https://globalprivacycontrol.org/' target='_blank'>globális adatvédelmi szabályozás</a>” és a „<a href='https://www.eff.org/issues/do-not-track' target='_blank'>ne kövessen</a>” jelzések küldése a weboldalaknak",
+        "description": "Checkbox label for enabling/disabling the Sec-GPC and DNT signals, found on the general settings page"
+    },
+    "options_disable_google_nav_error_service": {
+        "message": "Meglátogatott webcímek küldésének letiltása a Google számára. Ez letiltja a javaslatokat és a hasonló oldalakat, ha egy oldal nem található.",
```

Comment:
"suggestions and similar pages" or "suggestions for similar pages"? Should be "for". The suggestions *are* the similar pages, not a distinct thing.